frontend/apps/web/app/(mgmt)/[account]/transformers/EditTransformerOptions.tsx

```diff
@@ -55,15 +54,24 @@ interface Props {
 export default function EditTransformerOptions(props: Props): ReactElement {
   const { transformer, disabled, value, onSubmit } = props;
   const [isSheetOpen, setIsSheetOpen] = useState(false);
+
+  const handleClose = () => {
+    setIsSheetOpen(false);
+  };
+
   return (
-    <Sheet open={isSheetOpen} onOpenChange={setIsSheetOpen}>
+    <Sheet
+      open={isSheetOpen}
+      onOpenChange={(open) => open && setIsSheetOpen(open)}
```

Comment:
The way I'm reading this is that `onOpenChange` is changed to only work in a single direction now? So this will only fire setIsSheetOpen if `open` is true.

This seems unorthodox given the way the sheet is intended to be defined, so could you leave a short code comment explaining why it is being done this way? It seems like the onClick on :74 is handling the initial open, which would explain it.

This `EditTransformerOptions` is used in a few different spots and if I remember correctly, it only has troubles with auto-closing in the job mappings table. The bulk transformer sheet doesn't have the issue this PR is fixing.
![image](https://github.com/user-attachments/assets/88b11ae2-56f4-4e37-a8f4-8d604e94d438)

Were these changes tested in the places this thing is used to ensure it works as intended?

Reply:
Ah good catch yeah 74 sets it to be open so not need to set it on the onOpenChange, updated. 

The bulk transformer in the job mappings has the same problem. If you click outside of the form, it will auto-close. With this change, the bulk transformer, nosql and job mappings are all behaving like I want them to (not closing when you click outside of the form).